elastic/integrations
# Introduce kubernetes package granularity

## What does this PR do?
Changes `kubernetes` package to use input groups in order to add icons for each service to support granular search.
This PR follows approach used in https://github.com/elastic/integrations/pull/767.
This package follows the structure suggested at https://github.com/elastic/integrations/issues/1017#issuecomment-811824576.

## Checklist


## How to test this PR locally
1. Search for the integration in integrations search in Kibana
2. Check that all tiles are present
3. Select each tile and verify content is as expected.

## Related issues
- Closes https://github.com/elastic/integrations/issues/1017


## Screenshots
![Screenshot 2021-05-27 at 3 38 55 PM](https://user-images.githubusercontent.com/11754898/119827733-249c6080-bf02-11eb-9833-9aa3d8930971.png)
